src/components/Home/index.tsx

```diff
@@ -78,43 +82,29 @@ export default function Home({ recentPosts }: BlogProps) {
         <HomepageFeature title="About" compact={true}>
           <div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-4 xl:gap-6 w-full">
             <AboutCard title="Hardware" className="text-rose-500 fill-rose-500" Icon={CpuChipIcon}>
-              <p>Betaflight supports a wide range of flight controllers with at least an STM32F4 MCU</p>
-              <p>You can use nearly any piece of modern hardware out there. Full telemetry and high refresh rate for receivers, camera and VTX control, RGB LEDs, and much more!</p>
-              <p>Run the configurator on any device, whether it is using Windows, MacOS, Linux, or Android</p>
+              <p>Betaflight supports a wide range of flight controllers from a variety of manufacturers.</p>
+              <p>The 'Approved Partner' program provides hardware recommendations from the development team based.</p>
```

Comment:
Based? Typo or intended?

Reply:
the team is based

https://www.urbandictionary.com/define.php?term=based

Reply:
...yes it is a typo...must have left it in from an edit ;)

Reply:
Ok cleared that up...